include/cute/arch/xe_copy_2B.hpp

```diff
@@ -706,8 +738,11 @@ struct XE_2D_U16x32x32_LD_V {
 };
 
 struct XE_2D_U16x16x8_LD_T {
+  using BlockShape = Shape<_8, _16>;
   using inst_dtype = uint32_t;
```

Comment:
This is needed because we emulate intrinsic for other data types. Why should we do that? For Nvidia there seems to be a 1:1 mapping between a struct and a ptx instruction. I think we also should have a 1:1 mapping between struct and builtin.